<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerSettingsDialog</class>
 <widget class="QDialog" name="ServerSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>209</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>209</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RPi Server Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="serverSettingsGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Server IP Address:</string>
   </property>
   <widget class="QLabel" name="portnumberLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Number:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="addrBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>192</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="addrBox2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>168</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="addrBox3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="addrBox4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="portSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>5555</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="clientSettingsGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Client Settings:</string>
   </property>
   <widget class="QLabel" name="timeSampleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Sample Period:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondsLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="timeSampleSpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
  </widget>
  <zorder>serverSettingsGroup</zorder>
  <zorder>buttonBox</zorder>
  <zorder>clientSettingsGroup</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServerSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServerSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
